exporters/__init__.py

```diff
@@ -0,0 +1 @@
+import ozzy as exporters  # NOQA
```

Comment:
from what's here, I infer that `import exporters` won't give a warning, will it?